src/js/components/TextInput/doc.js

```diff
@@ -93,9 +94,10 @@ export const doc = TextInput => {
     placeholder: PropTypes.node.description(
       'Placeholder to use when no value is provided.',
     ),
-    plain: PropTypes.bool.description(
-      `Whether this is a plain input with no border or padding.
-      Only use this when the containing context provides sufficient affordance`,
+    plain: plainPropType.description(
+      `Whether this is a plain input with no border or outline.
+      Only use this when the containing context provides sufficient affordance.
+      Use "full" to remove padding in addition to removing border and outline.`,
```

Comment:
I think the context will make more sense if lines 2 & 3 could be flipped.

Reply:
Switched these lines in latest push